submissions/cumiOyemike/react/to-do-app/src/index.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  /* display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; */
```

Comment:
Remove commented out code

Reply:
Hello, I just made the changes. Kindly review again. Thanks.